core/db/default/spree/states.rb

```diff
@@ -21,8 +29,15 @@
 end.join(', ')
 
 state_values.call.each_slice(500) do |state_values_batch|
-  connection.execute <<-SQL
-    INSERT INTO spree_states (#{columns})
-    VALUES #{state_values_batch.join(", ")};
-  SQL
-end
+  if connection.adapter_name =~ /SQLite/i
+    connection.execute <<-SQL
+      INSERT INTO spree_states (#{columns})
+      SELECT #{state_values_batch.join(" UNION SELECT ")};
```

Comment:
Hi @Houndci, I don't get this line ***Prefer single-quoted strings inside interpolations.***
can you give an example? thanks much